docs/docs/references/tenants/README.md

```diff
@@ -0,0 +1,132 @@
+
+### I would like build a SaaS application with multi-tenancy
+
+If you are building a SaaS application with the concept of "workspace" or "organization" for each customer, you can use [organizations](/docs/recipes/organizations/) to manage each customer's workspace within a single tenant.
+
+```mermaid
+graph LR
+    subgraph Logto tenant
+        subgraph Users
+            U1[User 1]
+            U2[User 2]
+            U3[User 3]
+            U4[User 4]
+            U5[User 5]
+        end
+        subgraph Organizations
+            O1[Organization 1]
+            O2[Organization 2]
+            O3[Organization 3]
+        end
+
+        U1 --> O1
+        U2 --> O1
+        U2 --> O2
+        U3 --> O2
+        U3 --> O3
+        U4 --> O2
+        U5 --> O3
+    end
+```
+
+In this case, a user can be a member of multiple organizations. For example, a user can have a personal workspace and join the company's workspace.
+
+### I have multiple applications
+
+With Logto, you can manage multiple applications within a single tenant regardless of the application's type (web, mobile, desktop, etc.). Since Logto follows the [OpenID Connect](https://openid.net/connect/) and [OAuth 2.0](https://datatracker.ietf.org/doc/html/rfc6749) standards, you can integrate Logto with any application even if there's no Logto official SDK.
+
+## Tenant region
+
+When you create a tenant, you can choose the region where the tenant data is stored. It cannot be changed after the tenant is created. Here are the available regions:
+
+- EU (Amsterdam, Netherlands)
+- US (Arizona, United States)
+
+Usually, you should choose the region closest to your customers to minimize latency and improve performance.
+
+:::info
+Logto leverages the global edge network to deliver the best performance and availability for your applications. The request routing is optimized to ensure that your users are always connected with the best-performing option.
+:::
+
+## Tenant types
+
+There are two types of tenants in Logto Cloud.
+
+### Development
+
+The development tenant (dev tenant) is primarily intended for testing purposes and should not be utilized in a production environment. These tenants allow access to premium and paid features available in paid plans, free of charge and without requiring a subscription. However, there are certain [limitations](#limitations-of-development-tenants) that apply to development tenants.
+
+### Production
+
+The production tenant is where end-users access the live app and you might need a paid subscription. You can subscribe to the free, or pro plan to create a production tenant.
+
+:::note
+Please be aware that you can only specify your tenant types during creation; it is not possible to modify the tenant type once it has been created.
+:::
```

Comment:
Maybe we should put this before the development tenant paragraph (at L108). Putting it here makes me feel I could not upgrade my free plan to pro...

Reply:
cool updated